### PR TITLE
put add/remove columns btn to table column endding

### PR DESCRIPTION
**Pull request by Lijiaoa**
before:
image

after:

image

### CODE REVIEW

File: ts/webui/src/components/trial-detail/TableList.tsx [hunk 1]
```diff
@@ -368,16 +368,29 @@ class TableList extends React.Component<TableListProps, TableListState> {
...
+                key: '_operation',
+                fieldName: 'operation',
+                minWidth: 150,
+                maxWidth: 160,
+                isResizable: true,
+                className: 'detail-table',
+                onRender: this._renderOperationColumn.bind(this)
+            },
+            {
+                name: 'setting', // add/remove columns setting
```
Comment (ultmaster) [inline ts/webui/src/components/trial-detail/TableList.tsx:383]: Should it be `Setting`?

**Review by ultmaster — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

**Comment by Lijiaoa** [comment]
- [x] commit for adjust column width
for screen little:

image

for screen large:

image

- [x] commit for `setting icon font size`
- [x] add scss file changes for setting icon
- [ ] Looks like it's difficult to add `title attribute` for setting icon

File: ts/webui/src/components/trial-detail/TableList.tsx (hunk 1 above)
Comment (Lijiaoa) [reply ts/webui/src/components/trial-detail/TableList.tsx]: ok

**Comment by Lijiaoa** [comment]
Close for better design

**Pull request closed without merging**